ports/servo/glutin_app/window.rs

```diff
@@ -495,14 +495,21 @@ impl Window {
                 event: glutin::WindowEvent::MouseWheel { delta, phase, .. },
                 ..
             } => {
-                let (mut dx, mut dy) = match delta {
-                    MouseScrollDelta::LineDelta(dx, dy) => (dx, dy * LINE_HEIGHT),
+                let (mut dx, mut dy, mode) = match delta {
+                    MouseScrollDelta::LineDelta(dx, dy) => (dx as f64, (dy * LINE_HEIGHT) as f64, 0x01),
                     MouseScrollDelta::PixelDelta(position) => {
                         let position =
                             position.to_physical(self.device_hidpi_factor().get() as f64);
-                        (position.x as f32, position.y as f32)
+                        (position.x as f64, position.y as f64, 0x00)
```

Comment:
0x00 and 0x01 look like magic numbers. Can you use an enum?

Reply:
Yea, Will Fix

Reply:
@jdm the webidl file generates an enum for us in the `components/script` crate. I've tried importing to `window.rs` it with `use script::dom::bindings` and `use components::script::dom:: bindings` but both say "use of undeclared type or module". Is it possible to access this crate from the `ports/servo/glutin_app` directory?

Reply:
You will want to create an independent enum in components/script_traits and convert between it and the generated one inside the script/ code.